.rubocop.yml

```diff
@@ -965,3 +965,10 @@ RSpec/Rails/AvoidSetupHook:
   Description: Checks that tests use RSpec `before` hook over Rails `setup` method.
   Enabled: true
   StyleGuide: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/Rails/AvoidSetupHook
+
+RSpec/NoExpectationExample:
+  Enabled: false
+  AllowedPatterns:
```

Comment:
The version of rubocop-rspec currently on `main` is `2.13`, where `NoExpectationExample` is introduced, but is not configurable to allow other patterns. I actually think this is a bad cop, but for now, I have just bumped rubocop-rspec to `2.14` where it is at least configurable.